src/battle_script_commands.c

```diff
@@ -14844,12 +14844,16 @@ bool8 IsMoveAffectedByParentalBond(u16 move, u8 battlerId)
         {
             switch (GetBattlerMoveTargetType(battlerId, move))
             {
+            // Both foes are alive, spread move strikes once
             case MOVE_TARGET_BOTH:
-                if (CountAliveMonsInBattle(BATTLE_ALIVE_DEF_SIDE) >= 2) // Check for single target
+                if (IsBattlerAlive(gBattlerTarget) && IsBattlerAlive(BATTLE_PARTNER(gBattlerTarget)))
```

Comment:
I did a test with the original code and it worked for `MOVE_TARGET_BOTH`. I think I wasn't clear enough to say that the issue I was having was with `MOVE_TARGET_FOES_AND_ALLY` moves.